### ISSUE TITLE
Poetry  - Unable to find installation candidates for watchdog (2.3.0)

### ISSUE THREAD

**Issue by pedrorjbr** [gorakhargosh/watchdog#948]
```
• Installing watchdog (2.3.0)

  RuntimeError

  Unable to find installation candidates for watchdog (2.3.0)

  at /opt/hostedtoolcache/Python/3.10.10/x[64](https://github.com/carecode-health/nabla-integration/actions/runs/4263771539/jobs/7420937565#step:6:65)/lib/python3.10/site-packages/poetry/installation/chooser.py:105 in choose_for
      101│ 
      102│             links.append(link)
      103│ 
      104│         if not links:
    → 105│             raise RuntimeError(f"Unable to find installation candidates for {package}")
      106│ 
      107│         # Get the best link
      108│         chosen = max(links, key=lambda link: self._sort_key(package, link))
```

Is anyone facing the same issue?

**Comment by BoboTiG** [comment]
Well, all files are present here: https://pypi.org/project/watchdog/2.3.0/#files

I had such issues sometimes, like a cache not yet updated :thinking: 
Let's see if others hit the same issue. Please try again in a few hours :crossed_fingers:

**Comment by pedrorjbr** [comment]
![image](https://user-images.githubusercontent.com/8076399/221271916-db811e36-122b-44b1-b9da-37e4f1c50610.png)

**Comment by BoboTiG** [comment]
I don't think there is an issue on the watchdog end. Maybe should you see with Poetry directly?

**Comment by BoboTiG** [comment]
Is it solved with the 2.3.1 version?

**Comment by kurtmckee** [comment]
@pedrorjbr I use Windows and Powershell and might be able to reproduce.

Could you provide the output from these commands:

```ps1
Get-Command python
python --version

Get-Command poetry
poetry --version
```

I'd also like to confirm what happens if you wipe your pip cache. Would you try this and paste the entire output in?

```ps1
pip uninstall watchdog
# Run `pip uninstall watchdog` repeatedly until no older versions are found
pip uninstall watchdog

pip cache info
pip cache purge
pip cache info

pip install watchdog
```
